dask/templates/additional-worker-deployment.yaml

```
@@ -0,0 +1,165 @@
+{{- range $worker := .Values.additional_worker_groups }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "dask.fullname" . }}-worker
```

This should be `$` instead of `.` because we are inside a range.

This is why the `fullNameOverride` errors are happening in the CI.

```suggestion
  name: {{ template "dask.fullname" $ }}-worker
```

---

This also needs to be done everywhere else `.` is used on its own.

---

Yes, I caught that. Now I'm getting **Error: INSTALLATION FAILED: template: dask/templates/additional-worker-deployment.yaml:36:152: executing "dask/templates/additional-worker-deployment.yaml" at <8>: invalid value; expected string`**

---

It doesn't like something on line 36.